sympy/solvers/ode/single.py

```diff
@@ -185,6 +185,7 @@ class SingleODESolver:
     # Cache whether or not the equation has matched the method
     _matched = None  # type: Optional[bool]
 
+
     def __init__(self, ode_problem):
```

Comment:
Not sure why you've added a blank line here...

Reply:
That was unintentional. Might have been added while adding the flag `simplify`.